sdk/iot/common/inc/az_iot_common.h

```diff
@@ -35,6 +35,9 @@ enum
  */
 typedef enum
 {
+  // Default, unset value
+  AZ_IOT_STATUS_UNSET = 0,
```

Comment:
I would call this STATUS_UNKNOWN instead.

Reply:
Done